tests/nightly-kickoff.sh

```diff
@@ -0,0 +1,332 @@
+NIMBUS_USER=
+NIMBUS_PASSWORD=
+NIMBUS_GW=nimbus-gateway.eng.vmware.com
+SLACK_URL=https://hooks.slack.com/services/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
+GS_PROJECT_ID=eminent-nation-87317
+GS_CLIENT_EMAIL=vic-ci-logs@eminent-nation-87317.iam.gserviceaccount.com
+GS_PRIVATE_KEY="-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDmv6+b61zT/Wc/\n2ahOfIrjgKQxhucGizflqp0C520urDGWYB4qd1/W3MGP9z7rjmFVOeT2l6D+OcSw\nn7vvEzjDFce5OOlT1OAQTn1m5fqj7yurEEusMau6LQJpbf0Yv1IbpTI7BMIVjc0l\nvNvWV10mUZUlSHoYEkrMBkJE1vNZO961wjYH/nyWd2VI+Xx5bc4IK28SghPaFffQ\nAN9uTHGXdGaK/rsakfwA/hFEfEpCoaCqIzJTg4gdn6AfsjCXkKCNeMUVW2/xjpKW\nhEI3emgBuFZt3/GsdXGmF2MagVe/35r3oxCNHfDK3sgZUMDr/40RqNa0dgjF6OyD\nfJgeBmkXAgMBAAECggEBALyxE5GVLhvMpJn6Cz/jaFAR6jL19gXL4rHUiwDM7uOz\nu/kUMJbZd23kqARqUvGdRMrExQ9Bf01lQAqPFMe0GD6vmNtGRsde1LuA89spRYS5\nGCSS9s6g76UXGVnNr6KFEUe6FxFcGro1cwThI4RrfKjRHf2W/wCgNLoShC52+BiG\n5v7VpkZJUnrrUtESqd2GaOmOrMyJcyecTVuL1ODlGFtk9Cf4h+CpmTHWAJTBMvPv\nEBP0MjqyCcl8oBktNVzC0z53DvsUNhD6jgvfPZ+AboCebs0xtrtMDaooCvNQ4+2K\nuJoAG9saYyrLeKMULSSKH94+NUUvX2JkDGH2NJoXfgECgYEA/d7NWpGStMkkkI37\noeuZhqyn2n/KLC+aTG4PiWCGdwziul6Q0JlrrwGk+zFz6kNKmGim+mw3JCmHtcYF\n5GJra7CEAV6/e2tnhP2NY8d4wip/N/58xZhWuCSxHur/O8/TTN4tjJ4dIugcf5a3\nk0ZmCtAet2RWHroIqaMVEwAHhPcCgYEA6K86sHPJP/eF5YYdVB0rzDrhcYPSseSS\na/JtOd31F2QZhx654zMlfUjM5z+LVoisf9CntyAabCceGUALOtlGwkoWD8naqPLo\nk6l8bAdiTV5nK5oSGaKb8xC3JHOM2d89dT82NDPWDOCCwQvN4BFUjGtbBmtF6n8d\ncuI578+i1OECgYAidN4MX9u4m+BRmmO/21lQFRkHJ/cJvkBEBWAodihp+h6/ytv+\n5APgkemRimnALvft7a5UKOHnD5fyzPi5wb3wtNmF0hVNLAu12jAZjdZPDDOOJwVK\nUF3cymYb2ytfM9rrAPDPuBoeRcCwdIVgANsStqKko4Ko0vkgBRl0JbnfiwKBgQDD\nix9jUqr6WuXnsgHLwoggJgt3/jR+03xJw34Pd3yVn8XkS+okCcOjuh6Y6EoM+ucc\nsxl/SDdsVKNyzOOjHR3eAazwr85W1WynS3QIxVvTcVZ6ygwUBxfP+Wgv9fuUzYs0\nkV7YGAf24maAHY9ykp3fNAlXJ6emHhV9iqjt5C0PgQKBgASEqRg+lPo0a6r9e/cW\nRwYs/GNvBiZM3V2ttmwz7nyzRuOakcocFk/3ujz58cY2g4UOL/8Rm3ULqf2v+3U1\nDmT12CenkVHB6DE1fngiSUknQ6irf24XdvOiqUVCJAj3eFpmaFcN3NFL/iccGLjq\n8sTizN0defLu1MTOdgDrMzls\n-----END PRIVATE KEY-----\n"
```

Comment:
* Delete old `GS_PRIVATE_KEY` in Google Cloud console
* Generate new `GS_PRIVATE_KEY` in Google Cloud console
* Update plaintext `secrets.yml`
* Update `.drone.sec` with new `GS_PRIVATE_KEY`
* Use Drone secrets in place of this plaintext value